monitoring/uss_qualifier/scenarios/astm/utm/aggregate_checks.py

```
@@ -118,3 +124,73 @@ def _op_intent_details_step(self):
             )
+
+    def _confirm_test_harness_queries_work(self):
+        """
+        For each different type of call to the interoperability test instance,
+        we look for at least one successful query.
+        """
+        for participant, queries_by_type in self._attributed_queries.items():
+            self._validate_participant_test_interop_instance(
+                participant, queries_by_type
+            )
+
+    def _validate_participant_test_interop_instance(
+        self,
+        participant_id: str,
+        participant_queries: dict[QueryType, List[fetch.Query]],
+    ):
+        # Keep track of how many interactions we've found for this participant
+        # if there is None the condition is not met
+        test_interactions = 0
+        success_by_type: Dict[QueryType, bool] = {}
+        for query_type, queries in participant_queries.items():
+            if _is_interop_test_interaction(query_type):
+                test_interactions += len(queries)
+                success_by_type[query_type] = False
+                for query in queries:
+                    if 200 <= query.response.status_code < 300:
+                        success_by_type[query_type] = True
+                        break
+
+        self.record_note(
+            "test_interop_interactions",
+            f"Found {test_interactions} interactions with interoperability test instance for {participant_id}",
+        )
+        if test_interactions == 0:
+            self.record_note(
+                "test_interop_check_skipped",
+                f"Skipping check for {participant_id} because no interactions with "
+                f"interoperability test instance were found",
+            )
+            return
+
+        with self.check(
+            "Interoperability test instance is available", [participant_id]
+        ) as check:
+            if test_interactions == 0:
+                # TODO remove once finished with development
```

I'm not sure I follow this; what would be removed at completion of development?

---

I was referring to the `test_interactions == 0` condition and the call to `record_failed` that is guarded by it: given that if the condition is true we returned above already.

(The condition was there to fail noisily while I was building the scenario)